### PR TITLE
Moved image convert data source files to a single directory

### PR DESCRIPTION
**Pull request by kmilo9999**
I renamed the main folder to data_converter, and moved the raw-to-tiff and tiff-2-png files to the mentioned folder.
I believe the osom-data-converter should stay on a separate folder, due its resulting raw files can be loaded in the desktop volume viewer without any additional conversions.

### CODE REVIEW

**Review by mirestrepo — CHANGES_REQUESTED** [review]
There still seems to be a `raw_2_tiff` folder. Also, Move the `bosom_converter` also inside. If you are making a package, the submodules at the root level is not standard. 

Running the executables is not the issue. But if someone is every using this package from python directly, you would just want them to be able to say

`from volume_data_converter import osom_to_raw`. Your current set up doesn't allow that

I think `data_converter` is too general... maybe add a 3d or volume in there... we will need to rename the repo  later to match the package

**Comment by mirestrepo** [comment]
Also - in the `pyproject.toml` you'll want to match the package name so change the line
```
[tool.poetry]
name = "volume-data-converter"
```

other name ideas

`volume-conversion-utils`
`volume-convert`

**Comment by kmilo9999** [comment]
- Renamed `data_converter` to `volume_data_converter`
- Moved osom files to `volume_data_converter`
- Modified toml files reflecting changes in the folder structure
- Removed raw_2_tiff folder

File: volume_data_converter/conver_to_8_bit.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    
+    if os.path.isfile(src):
+         file_path, filename = os.path.split(dest)
+         file_name,file_ext = os.path.splitext(filename)
+
+         with open(src, "rb") as img_file:
+            img_data = np.fromfile(img_file, dtype=np.float32)
+            if img_data.dtype != np.uint8:
+                # scale to 8 bit
+                print("TODO")
```
Comment (mirestrepo) [inline volume_data_converter/conver_to_8_bit.py:69]: Don't forget this TODO :)

**Review by mirestrepo — APPROVED** [review]
Looks great! You left few TODOs that I imagine are coming later...

**Pull request merged**